Jenkinsfile

```diff
@@ -102,18 +102,12 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
         }
 
         stage('Fetch') {
-            // XXX: drop `!prod && ` once we've uploaded prod builds there
-            if (!prod && s3_builddir) {
```

Comment:
so we ran this once already? or do we need https://github.com/coreos/coreos-assembler/pull/545 first?

Reply:
We did `buildupload` once already, so we have something to `buildprep` from. https://github.com/coreos/coreos-assembler/pull/545 is for the case where there's nothing to `buildprep` from (which is mostly relevant for devel pipelines).